jobs/wf1-1-getTEIs.js

```diff
@@ -0,0 +1,13 @@
+// Get trackedEntityInstances from DHIS2
+// TODO: How to filter based on inactive?
+get(
+  'trackedEntityInstances',
+  {
+    ou: 'l22DQq4iV3G',
+    program: 'uGHvY5HFoLG',
+  },
+  state => {
```

Comment:
Just for my understanding, I don't think this callback is needed at all? It's not doing anything?

Reply:
It's taking data from `state.data` and put it in `state.trackedEntityInstances`